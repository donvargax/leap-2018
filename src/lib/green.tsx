import {h, Text} from "ink"

export const Green = ({children}: {children: (string | (string | Element)[])}) =>
  <Text green>{children}</Text>;