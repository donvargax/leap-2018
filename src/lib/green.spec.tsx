import { h, renderToString } from "ink"

import { Green } from "./green"

test('Should have Greeter available', () => {
  const result = renderToString(<Green>random</Green>)
  expect(result).toMatchSnapshot()
});
