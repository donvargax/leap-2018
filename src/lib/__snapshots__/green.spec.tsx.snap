// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have Greeter available 1`] = `"[32mrandom[39m"`;
